migrations/012.do.round-and-ip-address.sql

```diff
@@ -0,0 +1,15 @@
+-- bigint uses 64 bits, the upper limit is +9223372036854775807
+-- Postgres and SQL do not support unsigned integers
+ALTER TABLE retrievals ADD COLUMN created_at_round BIGINT;
+ALTER TABLE retrievals ADD COLUMN created_from_address INET;
+
+UPDATE retrievals SET created_at_round = 0, created_from_address = '0.0.0.0';
+ALTER TABLE retrievals ALTER COLUMN created_at_round SET NOT NULL;
+ALTER TABLE retrievals ALTER COLUMN created_from_address SET NOT NULL;
+
+ALTER TABLE retrieval_results ADD COLUMN completed_at_round BIGINT;
+ALTER TABLE retrieval_results ADD COLUMN completed_from_address INET;
```

Comment:
I am not sure yet whether we need to collect both "created" and "completed" round & address and how to handle the case when they are in conflict.

To make it easier to observe the real network of Stations, I prefer to collect them both, inspect the data and decide on the next steps based on that.

Reply:
I was wondering the same and came to the same conclusion 👍